homeassistant/components/zwave_js/strings.json

```
@@ -151,5 +151,193 @@
       "title": "Newer version of Z-Wave JS Server needed",
       "description": "The version of Z-Wave JS Server you are currently running is too old for this version of Home Assistant. Please update the Z-Wave JS Server to the latest version to fix this issue."
     }
+  },
+  "services": {
+    "clear_lock_usercode": {
+      "name": "Clear lock user code",
+      "description": "Clears a user code from a lock.",
+      "fields": {
+        "code_slot": {
+          "name": "Code slot",
+          "description": "Code slot to clear code from."
+        }
+      }
+    },
+    "set_lock_usercode": {
+      "name": "Set lock user code",
+      "description": "Sets an user code on a lock.",
```

How did this sneak in?
```suggestion
      "description": "Sets a user code on a lock.",
```

---

My mistake. I should have caught that.